actionview/lib/action_view/helpers/tags/base.rb

```diff
@@ -4,6 +4,8 @@ module ActionView
   module Helpers
     module Tags # :nodoc:
       class Base # :nodoc:
+        EMPTY_STRING = "".freeze
```

Comment:
Why the constant?

Reply:
Sure, I will update to match a single style